### ISSUE TITLE
Package lib for use in browsers

### ISSUE THREAD

**Issue by carloslenz** [fauna/faunadb-js#450]
Instructions say we should use the CDN version of the driver:

- https://docs.fauna.com/fauna/current/drivers/javascript.html
- https://github.com/fauna/faunadb-js

I tried to use the node.js version but it doesn't work (failing because util is not defined).  If I'm missing something, please update the docs.

Even if using CDNs is great for quick tests, SPAs are usually bundled with all dependencies included.

Please provide compatible packages so developers can use their usual workflow.

**Comment by fireridlle** [comment]
can you provide an example? 
if you use npm package and build application for web, your bundler tools should handle it

**Comment by faunaee** [comment]
This is a working example, provided that you edit it to use a valid secret:

```html
<html>
  <body>
    <h1>Test</h1>
    <p>
      Check the browser's developer tools' console for the result.
    </p>
  </body>

  <script src="//cdn.jsdelivr.net/npm/faunadb@latest/dist/faunadb.js"></script>
  <script type="text/javascript">

var faunadb = window.faunadb
var q = faunadb.query

var client = new faunadb.Client({
  secret: 'your secret',
})

;(async () => {
  var result = await client.query(
    q.Add(1, 1)
  )
  .then((ret) => ret)
  .catch((err) => console.log('Error:', err))

  console.log('Got result:', result)
})()
  </script>
</html>
```

**Comment by carloslenz** [comment]
I don't want to do this: `<script src="//cdn.jsdelivr.net/npm/faunadb@latest/dist/faunadb.js"></script>`

1. It's bleeding edge, unsuitable for production
2. Replacing `@latest` in an html file is not the way to upgrade in most SPAs. This is usually kept in `package.json`.

**Comment by carloslenz** [comment]
> can you provide an example?
> if you use npm package and build application for web, your bundler tools should handle it

Sure the bundler knows how to build it (I'm using rollup), but the code is using 'util', from node.js I presume.

So changing a working SPA from:

`const { Client, query } = (window as any).faunadb;`

To

`import { Client, query } from "faunadb";`

Results in `main.ts:18 Uncaught ReferenceError: util is not defined
    at main.ts:18` when the browser tries to run it.

**Comment by faunaee** [comment]
Which version of the driver were you using when you got the `util` is not defined error? To my knowledge, that issue has been fixed.

The point of the example HTML is that the CDN version *is* the Node.js version: it works in a browser.

**Comment by carloslenz** [comment]
> Which version of the driver were you using when you got the `util` is not defined error? To my knowledge, that issue has been fixed.

I was using 4.0.0, and indeed the latest version (4.1.2) is working.

> The point of the example HTML is that the CDN version _is_ the Node.js version: it works in a browser.

OK, I don't know when you fixed it but around 4.0.0 the CDN version worked for SPAs but npm package only worked for my console scripts.
